### PR TITLE
coala: Fix some code style problems in code

### PR DESCRIPTION
**Pull request by souravsingh**
Fixes codestyle problems in the coala codebase.

Fixes https://github.com/coala/coala/issues/2690

### CODE REVIEW

File: coalib/collecting/Collectors.py [hunk 1]
```diff
@@ -163,7 +163,8 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer,
     """
     bears_found = tuple([] for i in range(len(kinds)))
     bear_globs_with_bears = set()
-    for bear, glob in icollect_bears(bear_dirs, bear_globs, kinds, log_printer):
+    for bear, glob in icollect_bears(
+            bear_dirs, bear_globs, kinds, log_printer):
```
Comment (sils) [inline coalib/collecting/Collectors.py]: I don't see why this is better

**Comment by sils** [comment]
how did you make this change? Any tool involved?

**Comment by souravsingh** [comment]
I used the ``PEP8Bear`` as mentioned in the issue comment

**Comment by sils** [comment]
you didn't link any issue to this PR, I have no idea what this is about :P can you fix the gitmate issues and add a link to the commit message according coala.io/commits ? Be sure to amend your comit.

**Comment by souravsingh** [comment]
@sils Sorry for the bad commit message. I have forgotten about the contribution workflow for coala

**Comment by Mixih** [comment]
@souravsingh status update?

**Comment by souravsingh** [comment]
I am working on fixing the errors. Will push my changes by today.

**Comment by souravsingh** [comment]
@Mixih There seems to be some conflicts. How do I fix them?

**Comment by Mixih** [comment]
@souravsingh  Sorry about this really late reply. Uhm, that might be messy. Firstly check that they haven't been fixed in a new patch yet. Sencondly, try this link http://stackoverflow.com/questions/161813/how-to-resolve-merge-conflicts-in-git . If you get stuck, or want furthur guidance, ask on gitter.

**Comment by Nosferatul** [comment]
closing due to high inactivity

**Pull request closed without merging**